cli-output/src/cli_output.rs

```diff
@@ -2126,6 +2126,9 @@ impl fmt::Display for CliBlock {
         if let Some(block_time) = self.encoded_confirmed_block.block_time {
             writeln!(f, "Block Time: {:?}", Local.timestamp(block_time, 0))?;
         }
+        if let Some(block_height) = self.encoded_confirmed_block.block_height {
```

Comment:
heh, I foresee support issues when people now start running `solana block <BLOCK>` instead of `solana block <SLOT>`

Reply:
Totally. `solana block-by-height` in our future?

Reply:
`solana-ledger-tool slot ...` got it right though!  oh well :)